app/isomorphic/components/atoms/nav-bar-toggle-btn/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import PT from "prop-types";
+
+import "./nav-bar-toggle-btn.m.css";
+
+export const NavBarToggleBtn = ({ onMenuToggle, isMegaMenuOpen }) => {
+  return (
+    <button onClick={onMenuToggle} styleName={`hamburger ${isMegaMenuOpen ? "is-open" : ""}`}>
```

Comment:
Check click event on mobile
Rename it to Hambergermneu

Reply:
--done